test/io/testcjsonwriter.py

```diff
@@ -43,5 +43,11 @@ def test_cjson_generation(self):
         number_of_atoms = json_data['properties']['number of atoms']
         self.assertEqual(number_of_atoms, data.natom)
 
+        # Ensure the bond connectivity index starts from 0
+        bonds = json_data.get('bonds', None)
+        self.assertIsNotNone(bonds)
```

Comment:
This doesn't check that the index starts at zero. Consider what some example CJSON should look like:
``` json
  "bonds": {
    "connections": {
      "index": [
        0,
        1,
        0,
        2,
        0,
        3,
```

Reply:
Generally, I don't think the atom with index 0 is guaranteed to be involved in any bond.
That's why my all my test does is to make sure that the larges index used in bond connectivity doesn't exceed the number of atoms.

Reply:
While that is true, in the case of our test molecules, 0 will exist in the list, and should be checked, otherwise we aren't actually testing the change in the code. We need to know that it _can_ occur for some example.

Reply:
@berquist 
Ok, I also added a check to ensure that 0 is indeed part of the connections.